app/konnect/updates.md

```diff
@@ -9,6 +9,18 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## June 2024
+
+**API Requests**
+: {{site.konnect_short_name}} users have now access to a new feature that tracks [API requests](/konnect/analytics/api-requests/) in near real-time. API Requests provide detailed records for the requests that are made to your APIs. This information can not only help you understand your consumers better, but also simplifies any initial investigation into errors or performance issues by providing an intuitive web experience that is fully integrated into the rest of your {{site.konnect_short_name}} organization.
+
+**Manage control plane analytics data ingestion**
```

Comment:
What happens if I'm using analytics before this feature goes live, do I immediately start getting charged for it? Or is there a pop up in the UI with a warning letting me know I'll be charged? 

If there is no UI warning, I think we might want to be more direct in the changelog (though I think I wrote this one) that it will now cost them money, so they should disable it if they don't want to be charged.

Reply:
Consumption based users will be billed but they are already billed for this feature. 
Existing subscription users no
New customers yes.